core/indigo-core/molecule/src/base_molecule.cpp

```diff
@@ -4325,3 +4313,81 @@ void BaseMolecule::getBoundingBox(Rect2f& bbox) const
     }
     bbox = Rect2f(a, b);
 }
+
+void MetaData::addMetaObject(MetaObject* pobj)
```

Comment:
Why do we keep using raw pointers?

Reply:
Unfortunately we still can't use std classes as members in Indigo.